website/static/_redirects

```diff
@@ -1,3 +1,5 @@
+/v0.13/docs/building-a-new-adapter /guides/dbt-ecosystem/adapter-development/3-building-a-new-adapter 301
+
```

Comment:
I don't think you need to put this one all the way up here -- you could put it down on lines 648-650, just above the other similar ones. It's a bit confusing to have it here.

Reply:
ok will do.